include/vcpkg/paragraphparser.h

```diff
@@ -57,7 +57,7 @@ namespace vcpkg
         void required_field(StringView fieldname, std::pair<std::string&, TextRowCol&> out);
 
         std::string optional_field(StringView fieldname);
-        void optional_field(StringView fieldname, std::pair<std::string&, TextRowCol&> out);
+        void optional_field(StringView fieldname, std::pair<std::string&, TextRowCol&> out) const;
```

Comment:
This change is incorrect. `*this` doesn't change, but data `*this` controls does.

(Similarly, `T& vector::operator[](size_t)` doesn't change the vector, but it should not be `const`)